google-datacatalog-kafka-connector/tools/metadata_generator.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
```

Comment:
This is used only for performance testing, right? Perhaps it should be in the test directory, and / or mentioned in README

Reply:
I think I might put it into https://github.com/mesmacosta/google-datacatalog-connectors-profiler eventually - it is a Marcelo's repository where code for metadata generation for other connectors is stored. This is why this script is not mentioned in README. WDYT @mesmacosta?

Reply:
For now I'd add to the README, describing that it can be used for performance testing. 

Later if it makes sense move it to another repo, does it make sense?